client/securedrop_client/gui/auth/__init__.py

```diff
@@ -17,5 +17,6 @@
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
 
-# Import classes here to make possible to import them from securedrop_client.gui.auth
-from securedrop_client.gui.auth.dialog import LoginDialog  # noqa: F401
+from securedrop_client.gui.auth.dialog import LoginDialog
+
+__all__ = ["LoginDialog"]
```

Comment:
somehow this file got messed up in the merge, this reverts it to the state it is in in freedomofpress/0.10.2